view/adminhtml/templates/setup.phtml

```diff
@@ -13,14 +14,35 @@
             <div class="steps-buttons-col">
                 <div class="steps-button-title">
                     <h2><?= $block->escapeHtmlAttr(__('Step 1:')) ?></h2>
+                    <h4><?= $block->escapeHtmlAttr(__('We select the sector of our platform')) ?></h4>
+                    <div class="steps-button-container">
+                        <select id="sector-selector">
+                            <option value="" disabled<?php if(!$sector): ?> selected<?php endif; ?> ><?= $block->escapeHtmlAttr(__('Choose the sector of your ecommerce')) ?></option>
```

Comment:
In the future, the sectors are going to be requested to a custom endpoint.
So I think it would be better and cleaner generating the selector reading sectors data from an array in a loop.

Reply:
I've talked with enrique and álvaro and this task will be performed in the near future so it is not worth it changing it.

Reply:
Ok